RepoCleanup/Application/Commands/AddTeamToRepoCommand.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace RepoCleanup.Application.Commands
+{
+    public class AddTeamToRepoCommand
```

Comment:
I see I forgot to create an ICommand interface, but we can do that later to further polish the C-part in the CQRS pattern if we decide to play around with this.